File: app/src/main/java/com/aa/androidormcomparison/measures/MeasureDatabase/MeasurementsDatabase.java
package com.aa.androidormcomparison.measures.MeasureDatabase;

import androidx.room.Database;
import androidx.room.RoomDatabase;
import androidx.room.TypeConverters;

@Database(version = 1, entities = {Measurement.class}, exportSchema = false)
@TypeConverters(DatesConverter.class)
public abstract class MeasurementsDatabase extends RoomDatabase {
    public abstract MeasurementDao measurementDao();
}

File: app/src/main/java/com/aa/androidormcomparison/measures/MeasureDatabase/MeasurementsDatabase.kt
package com.aa.androidormcomparison.measures.MeasureDatabase

import androidx.room.Database
import androidx.room.RoomDatabase

@Database(entities = [Measurement::class], version = 1, exportSchema = false)
abstract class MeasurementsDatabase : RoomDatabase() {
    abstract fun measurementDao(): MeasurementDao
}